docs/usage/configuration-options.md

```diff
@@ -255,6 +255,10 @@ Add config here if you wish it to apply to Docker package managers Dockerfile an
 
 ## dotnet
 
+## draftPR
+
+If `draftPR` is set to true, the PR created by Renovate is set to a draft. Currently only available in Github.
```

Comment:
This text shouldn't duplicate the text in the config/definitions description. Ideally rewrite this to include a link to draft PRs documentation on github.com

Reply:
Possibly fixed by https://github.com/renovatebot/renovate/pull/6390/commits/f9cb3477367f6951b75665c26347ba55919b5ded